backend-models/service-tests/courseedservice-test.txt

```diff
@@ -0,0 +1,1284 @@
+courseedservice.php
+
+---------------------------------------------------------------------------------------------------------------------
+
+========================================================================
+========================================================================
+===                           New Course                             ===
+========================================================================
+========================================================================
+
+*********
+* pre-req:
+********* 
+
+if(checklogin()){
+	$isSuperUserVar=isSuperUser($userid);
+
+	$ha = $isSuperUserVar;
+
+	if($ha){
+
+$query = $pdo->prepare("INSERT INTO course (coursecode,coursename,visibility,creator, hp, courseGitURL) VALUES(:coursecode,:coursename,0,:usrid, 7.5, :courseGitURL)");
```

Comment:
The query values is to be supplied as actual input-values in an array.

Example:
///////
/ pre-req variables
//////

$pre-Values = { null,
                1885, 
                0,
                1,
                "AutomaticTest",
                "Quiz",
                "2023-04-27 00:00:00",
                "2023-04-29 00:00:00",
                2,
                "1337",
                "2023-04-17",
                "{&quot;deadline1&quot;:&quot;2023-04-27 0:0&quot;,&quot;comment1&quot;:&quot;&quot;,&quot;deadline2&quot;:&quot;&quot;,&quot;comment2&quot;:&quot;&quot;,&quot;deadline3&quot;:&quot;&quot;,&quot;comment3&quot;:&quot;&quot;}", 
                0 }; 


SQL-query:  INSERT INTO quiz(cid,autograde,gradesystem,qname,quizFile,qrelease,deadline,creator,vers,qstart,jsondeadline,`group`) 
            VALUES ($pre-Values[1], $pre-Values[2], $pre-Values[3], $pre-Values[4], $pre-Values[5], $pre-Values[6], $pre-Values[7], $pre-Values[8], $pre-Values[9], $pre-Values[10], $pre-Values[11] );
            
the used values need to be correct, you should be able to see the added course in LenaSYS